AuthenticationTests/LogIn/LoginViewModelTests.swift

```diff
@@ -36,12 +36,36 @@ class LoginViewModelSpec: QuickSpec {
                 expect(loginVM.passwordVisible.value) == false
             }
             
+            describe("#togglePasswordVisibility") {
+                
+                context("when executing action") {
+                    
+                    it("should change #passwordVisible from false to true") { waitUntil { done in
```

Comment:
I would move the false and true part to the context:
```swift
context("When executing action") {
    context("When #passwordVisible is false") {
        it("should change it to true") {
        }
    }
    context("When #passwordVisible is true") {
        it("should change it to false") {
        }
    }
}
```
However, you decide, it isn't so important so if you want to leave it like this its okay